README.md

```diff
@@ -14,8 +14,15 @@ fanciful decoration.
 
 # example
 
+Example files: [example/parse.js](./example/parse.js) (CommonJS) / [example/parse.mjs](./example/parse.mjs) (ECMAScript)
```

Comment:
```suggestion
Example files: [example/parse.js](./example/parse.js) (CJS) / [example/parse.mjs](./example/parse.mjs) (ESM)
```
(since "CommonJS" is actually a bit broader than CJS, and "ECMAScript" is much broader than ESM)